include/gridtools/common/defs.hpp

```diff
@@ -22,7 +22,11 @@
    @brief global definitions
 */
 
+#ifdef __CUDACC__
+#define GT_CONSTEXPR
```

Comment:
If we do this with `constexpr` / `wstd` I would prefer this being in a separate PR because obviously that is a likely candidate for performance regressions...